hongit-api/src/main/kotlin/com/devfloor/hongit/api/article/application/ArticleService.kt

```diff
@@ -77,6 +80,42 @@ class ArticleService(
             }
     }
 
+    fun showTopFiveByFavorite(): List<ArticleHomeResponse> {
+        return articleRepositoryCustom.findByFavoriteTopFive()
+            .map { article ->
+                val articleFavorites = articleFavoriteRepository.findAllByArticle(article)
+
+                ArticleHomeResponse(
+                    article = article,
+                    articleFavorites = articleFavorites,
+                )
+            }
+    }
```

Comment:
간결함을 위해서 이런 코드는 어떨까요?
만약 괜찮을 것 같다면, 이 메서드 뿐 아니라 비슷한 구조를 갖는 다른 `showTopFive~~` 메서드들도 마찬가지로 간결하게 바꿔도 좋을거같아요 !

```suggestion
    fun showTopFiveByFavorite(): List<ArticleHomeResponse> =
        articleRepositoryCustom.findByFavoriteTopFive()
            .map {
                ArticleHomeResponse(
                    article = it,
                    articleFavorites = articleFavoriteRepository.findAllByArticle(it),
                )
            }
```